app/helpers/host_helper/textual_summary.rb

```diff
@@ -235,6 +235,10 @@ def textual_cluster
     h
   end
 
+  def textual_physical_server
+    {:label => _("Physical Server"), :value => @record.physical_server.try(:name), :icon => "pficon pficon-server", :link => url_for(:controller => 'physical_server', :action => 'show', :id => @record.physical_server_id)}
```

Comment:
@GabrielSVinha Not all hosts are expected to have a Physical Server associated with them, therefore, I recommend adding a conditional here.
```ruby
{:label => _("Physical Server"), :value => @record.physical_server.try(:name), :icon => "pficon pficon-server", :link => url_for(:controller => 'physical_server', :action => 'show', :id => @record.physical_server_id)} if @record.physical_server_id
```